lib/sequent/support/database.rb

```diff
@@ -0,0 +1,63 @@
+module Sequent
+  module Support
+    # Offers support operations for a postgres database.
+    #
+    # Class methods do establish their own database connections (and therefore
+    # take in a database configuration). Instance methods assume that a database
+    # connection yet is established.
+    class Database
+      attr_reader :db_config
+
+      def self.create!(db_config)
+        ActiveRecord::Base.establish_connection(db_config.merge('database' => 'postgres'))
+        ActiveRecord::Base.connection.create_database(db_config['database'])
+      end
+
+      def self.drop!(db_config)
+        ActiveRecord::Base.establish_connection(db_config.merge('database' => 'postgres'))
+        ActiveRecord::Base.connection.drop_database(db_config['database'])
+      end
+
+      def self.establish_connections(db_config, view_schema_name)
+        Sequent::Support::BaseViewModel.establish_connection(
```

Comment:
As far as I can tell this would only affect consistency when view models influence consistency of the domain, which I think they shouldn't?

That said, it might be better to provide more control or configuration on this.